framework/helpers/BaseConsole.php

```diff
@@ -340,6 +340,93 @@ public static function ansiStrlen($string)
         return mb_strlen(static::stripAnsiFormat($string));
     }
 
+    //
+    // Exit code constant names, values, and descriptions are taken from
+    // OpenBSD's sysexits(3) man page http://man.openbsd.org/sysexits
+    //
+    // The command completed successfully.
+    const EX_OK = 0;
```

Comment:
Would that a better home for the rest of this?

(In general I have a growing preference for helpers, or, iow, not requiring objects when there's no need. Probably because conventions for namespaces and autoloading have stabilized.)